File: Chapter 3/Java/PIWebAPIJavaExample/src/Application.java
import java.util.List;

public class Application {
	public static void main(String[] args) 
	{
	List<String> citiesNames = PIWebAPIWrapper.GetCitiesName();
    List<String> attributesNames = PIWebAPIWrapper.GetAttributesName();
    String value = "1320";
    int statusCode = PIWebAPIWrapper.SendValue(citiesNames.get(0), attributesNames.get(1), value);
  
	    if (statusCode == 202)
	    {
	    	System.out.println("Success!");
	    }
	    else
	    {
	    	System.out.println("Error");
	    }
	}
}

File: Chapter 2/Matlab/Application.m
clear all;
close all;
clc;
addpath('C:\Program Files\MATLAB\jsonlab');
citiesName =PIWebAPIWrapper.GetCitiesName();
attributesName = PIWebAPIWrapper.GetAttributesName();
value=50;

statusCode = PIWebAPIWrapper.SendValue(citiesName{1}, attributesName{2}, value);
if (statusCode==202)
    display('Success!');
else
    display('Error!');
end